source/core/ib_stencil.cc

```diff
@@ -149,35 +213,94 @@ IBStencil<dim>::point_for_cell_detection(IBParticle<dim> & p,
   Point<dim> surface_point;
   p.closest_surface_point(dof_point, surface_point);
   Tensor<1, dim, double> vect_ib = dof_point - surface_point;
-  Point<dim>             point   = dof_point + vect_ib * 1.0 / 16;
+  Point<dim>             point   = dof_point + vect_ib / 16.;
 
   return point;
 }
 
+template <int dim>
+double
+IBStencil<dim>::ib_velocity(
+  IBParticle<dim> &                                     p,
+  const Point<dim> &                                    dof_point,
+  unsigned int                                          component,
+  const typename DoFHandler<dim>::active_cell_iterator &cell_guess)
+{
+  double v_ib = 0;
```

Comment:
What is v_ib? can you use a more verbose variable name?

Reply:
I renamed it to immersed_boundary_velocity, I think that's better